sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -30,7 +26,7 @@ To try and improve the development experience across Azure services, including S
 
 The modern Service Bus client library provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new `Azure.Identity` library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. 
 
-While we believe that there is significant benefit to adopting the modern version of the Service Bus library, it is important to be aware that the legacy version has not been officially deprecated. It will continue to be supported with security and bug fixes as well as receiving some minor refinements. However, in the near future it will not be under active development and new features are unlikely to be added. There is no guarantee of feature parity between the modern and legacy client library versions.
+While we believe that there is significant benefit to adopting the modern version of the Service Bus library, it is important to be aware that the legacy version has not been officially deprecated. It will continue to be supported with security and bug fixes as well as receiving some minor refinements. However, in the near future it will not be under active development and new features are unlikely to be added.
```

Comment:
Not sure that I agree with the removal in this paragraph.   Would you be kind enough to share the rationale behind this change?

Reply:
I see this statement giving a knee jerk reaction to someone reading it. 

Am guessing the intention here was that moving forward, the modern library would have features that are not guaranteed to be in the legacy version?
My concern is that this can be misunderstood to be the other way around i.e. the modern library will not have the same features as the legacy one

Reply:
Two points to consider. When you say "legacy client" there are two of those. Be specific or elaborate. The statement should be there to warn about breaking changes. If someone relies on the pipelines, they won't be able to move to track 2 SDK as it has no support for message pipelines. At no point it should be a surprise.

Reply:
Improved in https://github.com/Azure/azure-sdk-for-net/pull/14166/commits/26be38e6a9a72f2d16b6132ce0aad5617010c374 by referencing the SDKs by name